Converters/CSi/Speckle.Converters.CSiShared/ToSpeckle/Geometry/LineToSpeckleConverter.cs

```diff
@@ -19,17 +29,17 @@ ITypedConverter<CSiJointWrapper, Point> pointConverter
     _pointConverter = pointConverter;
   }
 
-  public Line Convert(CSiFrameWrapper target) // NOTE: THIS IS TEMPORARY POC
+  public Line Convert(CSiFrameWrapper target)
   {
-    // frame points
+    // TODO: Better exception handling
```

Comment:
ideally you should not necessarily handle exceptions here in typed converters. We handle conversion exceptions on RootObjectBuilders to report them UI. see [here ](https://github.com/specklesystems/speckle-sharp-connectors/blob/6ba70a76c2eb7205f911613adf8ea4b697242db1/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RevitRootObjectBuilder.cs#L132). But not sure you have special cases around line, let me know if so, i would like to know

Reply:
In referene to Etabs' API return for an queries is an `int` if it was successfull or not. All of these should be returning 0 (CSi's way of saying succeeded). Currently, this is handled in the `if` blocks.